frontend/src/modules/scribble/components/Timer.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="timer">{{ timeLeft }}</div>
+</template>
+
+<script>
+export default {
+    name: "Timer",
+    data() {
+        return {
+            timerInterval: null,
+            timePassed: 0,
+        };
+    },
+    props: {
+        startTime: Number,
+    },
+    computed: {
+        timeLeft() {
+            return this.timeInSeconds(this.startTime) - this.timePassed;
+        },
+    },
+    methods: {
+        startTimer() {
+            this.timerInterval = setInterval(
+                () => (this.timePassed += 1),
+                1000
+            );
+        },
+        timeInSeconds(nanoseconds) {
```

Comment:
Should we just have the start time in seconds instead of nanoseconds?

Reply:
If we decide to keep using nanoseconds, this can also be put into exponential format for readability, ie 1eX